editor/src/com/talosvfx/talos/editor/notifications/Notifications.java

```diff
@@ -36,6 +33,22 @@ public interface EventRunner {
     public interface Event extends Pool.Poolable {
         @Override
         void reset ();
+
+        default boolean notifyThroughSocket () {
+            return false;
+        }
+
+        default String getEventType () {
+            return "";
+        }
```

Comment:
Maybe create an Event abstract class, make all events inherit from this one, and have the methods there?
I just don't like having default methods in interfaces